README.md

```diff
@@ -39,7 +39,11 @@ module.exports = {
 };
 ```
 
-This extracts SourceMaps from all js files (including node_modules). This is not very performant, so you may want to only apply the loader to relevant files.
+`source-map-loader` extracts existing [sourcemaps](http://blog.teamtreehouse.com/introduction-source-maps) from all JavaScript entries.  This includes both inline sourcemaps as well as map data linked via URL. All map data is passed to WebPack for processing as per a chosen [source mapping style](https://webpack.js.org/configuration/devtool/) specified by the `devtool` option in [webpack.conf.js](https://webpack.js.org/configuration/).
```

Comment:
` from all JavaScript entries.  <= This includes ` => ` from all JavaScript entries. This includes` (Spacing)